src/Player.js

```diff
@@ -1,5 +1,15 @@
 /* Represents a player in the game; either computer-controlled or human-controlled */
 class Player 
 {
+	constructor(ownedPieces, capturedPieces)
+	{
+		this.ownedPieces = ownedPieces;
```

Comment:
Does this refer to the pieces that are present on the board or pieces that can be dropped (like the tower example)?